packages/db/src/lib/execution.ts

```diff
@@ -61,33 +49,37 @@ export const getExecutionInfoByGranuleCumulusId = async ({
 };
 
 /**
- * Returns a list of executionArns sorted by most recent first, for an input
+ * Returns execution info sorted by most recent first for an input
```

Comment:
Nit: Grammar improvement 👍🏻  , not sure this shouldn't be `execution records` though.     Nit because it's largely irrelevant if the typing is correct.

Reply:
https://github.com/nasa/cumulus/pull/3853/commits/bb18a572f64ced69253a94d8728d758c94741195